### PR TITLE
Update Verify content based on Patrick’s feedback

### PR DESCRIPTION
**Pull request by tekin**

### CODE REVIEW

File: app/views/verify/authentications/failed.html.erb [hunk 1]
```diff
@@ -2,22 +2,35 @@
...
     </p>
 
     <ul class="govuk-list govuk-list--bullet">
       <li>
-        your answer to a question doesn't match the records held by the Verify provider
+        the company you chose does not have enough information about you
       </li>
       <li>
-        the Verify provider that you chose does not have access to enough information about you
+        the company’s records are not up to date
```
Comment (pezholio) [inline app/views/verify/authentications/failed.html.erb]: Is it clear that `the company` referred to here refers to `the company you chose`, or should we repeat this?

File: app/views/verify/authentications/failed.html.erb [hunk 2]
```diff
@@ -2,22 +2,35 @@
...
+    <h2 class="govuk-heading-m">
+      Next steps
+    </h2>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        check your answers are correct
+      </li>
+      <li>
+        try using a different company
```
Comment (pezholio) [inline app/views/verify/authentications/failed.html.erb]: `try using a different company for verification` maybe?

File: app/views/verify/authentications/failed.html.erb (hunk 1 above)
Comment (PatrickTaylorDfE) [reply app/views/verify/authentications/failed.html.erb]: We call it the company you chose up above this, so though I considered it, repeating the 'you chose' seemed a bit excessive to me.

File: app/views/verify/authentications/failed.html.erb (hunk 2 above)
Comment (PatrickTaylorDfE) [reply app/views/verify/authentications/failed.html.erb]: If we were going to do this then I would probably go for 'try choosing a different company in GOV.UK Verify'

File: app/views/verify/authentications/failed.html.erb (hunk 1 above)
Comment (pezholio) [reply app/views/verify/authentications/failed.html.erb]: Cool fair enough 👍

File: app/views/verify/authentications/failed.html.erb (hunk 2 above)
Comment (tekin) [reply app/views/verify/authentications/failed.html.erb]: @PatrickTaylorDfE do you have a preference either way?

Comment (PatrickTaylorDfE) [reply app/views/verify/authentications/failed.html.erb]: My preference would be to leave as is.

Comment (pezholio) [reply app/views/verify/authentications/failed.html.erb]: Cool, let's leave it then. I'm probably overthinking it 😄

**Pull request merged**